templates/k-bims-ledet/0/docker-compose.yml

```diff
@@ -0,0 +1,362 @@
+# Base docker-compose file for rancher in staging environment
+# This file has some specific tweaks for staging environment
+# You can copy this file and paste it when creating a new stack in rancher
+# It will try to sync related files
+
+# Rancher currently only supports up to version 2
+# If you want to use docker-swarm, create another orchestration with version 3
+version: '2'
+volumes:
+  postgis-data:
+  db-backups:
+    #driver: local
+    #driver_opts:
+    #  type: cifs
+    #  o: vers=3.02,mfsymlinks,username=${SMB_USER},password=${SMB_PASSWORD}
+    #  device: "${SMB_PATH}/k-bims/db-backups/"
+  django-media:
+    #  driver: local
+    #  driver_opts:
+    #    type: cifs
+    #    o: vers=3.02,mfsymlinks,username=${SMB_USER},password=${SMB_PASSWORD}
+    #    device: "${SMB_PATH}/k-bims/media/"
+  django-media-backups:
+    #  driver: local
+    #  driver_opts:
+    #    type: cifs
+    #    o: vers=3.02,mfsymlinks,username=${SMB_USER},password=${SMB_PASSWORD}
+    #    device: "${SMB_PATH}/k-bims/media-backups/"
+  django-static:
+    #  driver: local
+    #  driver_opts:
+    #    type: cifs
+    #    o: vers=3.02,mfsymlinks,username=${SMB_USER},password=${SMB_PASSWORD}
+    #    device: "${SMB_PATH}/k-bims/static/"
+  django-reports:
+  django-logs:
+  nginx-logs:
+  nginx-conf-data:
+  geoserver-data-dir:
+
+services:
+
+  smtp:
+    image: catatnight/postfix
+    environment:
+      maildomain: kartoza.com
+      smtp_user: noreply:docker
+    restart: unless-stopped
+
+  nginx-conf-data:
+    image: kartoza/ledet_nginx_conf
+    restart: unless-stopped
+    labels:
+      io.rancher.container.start_once: true
+    volumes:
+      - nginx-conf-data:/etc/nginx/sites-available
+
+  lb:
+    image: rancher/lb-service-haproxy:v0.7.15
+    ports:
+    - ${WEBSERVER_PORT}:80/tcp
+    labels:
+      io.rancher.container.agent.role: environmentAdmin,agent
+      io.rancher.container.agent_service.drain_provider: 'true'
+      io.rancher.container.create_agent: 'true'
+
+  web:
+    image: nginx
+    volumes:
+    - django-static:/home/web/static:ro
+    - django-media:/home/web/media:ro
+    - nginx-conf-data:/etc/nginx/sites-available
+    - nginx-logs:/var/log/nginx
+    entrypoint: /etc/nginx/sites-available/docker-entrypoint.sh
+    links:
+      - uwsgi:uwsgi
+      - geoserver:geoserver
+    command: prod
+    labels:
+      io.rancher.container.pull_image: always
+    restart: unless-stopped
+
+  data-dir-conf:
+    image: geosolutionsit/geoserver_data:2.13.x
+    restart: unless-stopped
+    labels:
+      io.rancher.container.start_once: true
+    volumes:
+      - geoserver-data-dir:/geoserver_data/data
+
+  geoserver:
+    image: geosolutionsit/geoserver-docker:2.13.x
```

Comment:
Can you check in with @NyakudyaA on this? I would prefer that we use the Kartoza GeoServer image everywhere because we have applied a lot of security fixes etc. to it and patched in support for extra drivers and so on.

Reply:
I see, but it's possible that it's not going to be smooth transition though. For example, in this docker image, they had several entrypoint script to set the password and geoserver data dir, probably more.